client/src/components/judging/JudgingRoom.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import roomData = require('../../data.json');
+
+
+interface JudgingRoomProps {
+    table: string;
+}
+
+// let styles = {
```

Comment:
Trim styles if not used. Also, `const` over `let` (and maybe freeze it to, but that's whatever)